docs/main/presentations.md

```diff
@@ -1,9 +1,6 @@
 ---
 title: Presenting Pi-hole
 description: Support for presenters
-last_updated: Sun Jan 13 18:37:35 2019 UTC
+last_updated: Sun Feb 9 2020

```

Comment:
The last_updated change is probably moot; you can remove the whole line

Reply:
I'd like to keep it. I know there's a PR to auto add the date of last change but I prefer to also keep yaml frontmatter.

Reply:
It's moot since the page won't (or shouldn't be included) in the sitemap.

Reply:
/shrug. It stays for now.

Reply:
It's totally redundant... the page won't be in sitemap since it's redirected.

Also, you need to remove the page from the nav config

Reply:
NVM about the last part, it's not in nav

Reply:
https://github.com/pi-hole/docs/pull/241/files#diff-45ee620855dad847bea76b49932ad7dbL52 ?

Reply:
For now it stays.